Loenn/entities/electrified_wall.lua

```diff
@@ -0,0 +1,78 @@
+local utils = require("utils")
+
+local fillColor = {0.55, 0.97, 0.96, 0.4}
+local borderColor = {0.99, 0.96, 0.47, 1.0}
+local fillColorInactive = {0.55, 0.97, 0.96, 0.1}
+local borderColorInactive = {0.99, 0.96, 0.47, 0.25}
+
+local function createHandler(name, dir)
+    local horizontal = dir == "up" or dir == "down"
+
+    local handler = {}
+
+    handler.name = name
+    handler.depth = -1
+    handler.canResize = {horizontal, not horizontal}
+
+    local minWidth, minHeight = horizontal and 16 or 8, horizontal and 8 or 16
+
+    handler.minimumSize = {minWidth, minHeight}
+
+    handler.placements = {
+        {
+            name = "active",
+            data = {
+                width = minWidth,
+                height = minHeight,
+                activationId = "",
+                startActive = true
+            }
+        },
+        {
+            name = "inactive",
+            data = {
+                width = minWidth,
+                height = minHeight,
+                activationId = "",
+                startActive = false
+            }
+        }
+    }
+
+    function handler.rectangle(room, entity, _)
+        return handler.selection(room, entity)
+    end
+    function handler.fillColor(room, entity)
```

Comment:
These rendering functions make the spikes look almost identical to the vanilla lightning in Loenn which is confusing. Might want to render the spikes as well like in the Ahorn plugin

Reply:
makes the 4 handlers way harder to condense into one file, but i'll work on that. could also just split them into four files like i did for pistons (except i might recondense piston plugins).